languages/python/oso/tests/test_polar_roles.py

```diff
@@ -335,11 +348,16 @@ def test_relationship_without_resources(init_oso):
     parent_child(parent_org, repo: Repo) if
         repo.org = parent_org and
         parent_org matches Org;
+
+    actor_has_role_for_resource(_, _, _);
     """
     oso.load_str(policy)
 
-    with pytest.raises(OsoError):
+    with pytest.raises(OsoError) as e:
         oso.enable_roles()
+    assert e.match(
+        "Need to define at least one `resource/4` predicate to use oso roles"
```

Comment:
I know we use this syntax internally a lot and it's a common prolog thing `rulename/num_args` but I don't think we ever talk like this in the docs so could be confusing to people with no prior prolog knowledge.

Reply:
We also don't really say predicate in the docs, we call them rules.

Reply:
Yeah ik, I could write out the arg names it's just kind of long. But prob a good idea.